specification/billing/resource-manager/Microsoft.Billing/preview/2019-10-01-preview/billing.json

```diff
@@ -5352,7 +5352,7 @@
             "modelAsString": true
           }
         },
-        "customerType": {
+        "accountType": {
```

Comment:
>accountType [](start = 9, length = 11)

This is a breaking changes. Can you introduce this in a new API version?

Reply:
Added customerType back. accountType  will be a new property. Will remove customerType in next version.

Reply:
Makes sense. Thanks for fixing...

---
In reply to: [370401325](https://github.com/Azure/azure-rest-api-specs/pull/8202#discussion_r370401325) [](ancestors = 370401325)